cfme/tests/test_db_migrate.py

```diff
@@ -116,7 +158,10 @@ def download_and_migrate_db(app, db_url):
 
 @pytest.mark.ignore_stream('upstream')
 @pytest.mark.tier(2)
-@pytest.mark.meta(automates=[1734076])
+@pytest.mark.meta(automates=[1734076, 1755553])
+@pytest.mark.uncollectif(
+    lambda appliance, db_version: appliance.version >= '5.10' and Version(db_version) < '5.6',
```

Comment:
Q: should appliance be or `temp_appliance_extended_db` or does it not matter?

Reply:
All the tests are passing => Doesn't seem to matter.

Reply:
:grin: good way to think about it

Reply:
@john-dupuy oh I see the question was more like "why did you use extended db" so:
1) I didn'T change what apparently used to work. I think non-extended DB will not be large enough.
2) Rude things are happening to the appliance in this test. I am not sure we want to use the precious `appliance` and risk causing bad things happening later (like stale cache entries on various places imaginable in the test process memory structures).
3) Also If we use `appliance`, The appliance DB has to be wiped after this test anyway as the db dumps this test is restoring contains various stuff in various params combination.